src/main/java/com/google/sps/app/servlets/GetTrips.java

```diff
@@ -21,30 +21,38 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-@WebServlet("/get-trip")
-public class GetTrip extends HttpServlet {
+@WebServlet("/get-trips")
+public class GetTrips extends HttpServlet {
 
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
         String userID = request.getParameter("userID");
-        List<Trip> Trip = getTrip(userID);
+        List<Trip> Trip = getTrips(userID);
```

Comment:
Maybe rename Trip to Trips too indicate that you'll be expecting to return multiple.